monitoring/ceph-mixin/README.md

```diff
@@ -0,0 +1,40 @@
+## Ceph mixin
```

Comment:
I see several variations of the name:

1. `Ceph mixin`
2. `ceph-mixin`
3. `the ceph-mixin`

Which one is "the real McCoy"? I suggest to choose one variant and stick to it. Having multiple variants of a proper name is problematic! 

If it is a proper name, there should probably not be any "the" before it - unless of course the "the" is actually a part of the name (as in, e.g., "The Thing") . . . but in that case the "the" should appear *everywhere*.

If the name is `ceph-mixin` and it appears as the first word of a sentence, it should probably be capitalized.

Reply:
I no longer refer to the term `ceph-mixin`. I took some inspiration of the readme of [kubernetes-mixin](https://github.com/kubernetes-monitoring/kubernetes-mixin) and rook's [ceph-mixin](https://github.com/ceph/ceph-mixins).